openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorConstants.java

```diff
@@ -370,6 +370,8 @@ public String getValue() {
     public static final String YML_EXTENSION = ".yml";
     public static final String JSON_EXTENSION = ".json";
     public static final String UNSUPPORTED_OPENAPI_VERSION_PARSER_MESSAGE = "attribute swaggerVersion is unexpected";
+    public static final String SYNTAX_ERROR_STRING_PATTERN = "WARNING: Error snake-parsing yaml content";
+    public static final String SYNTAX_ERROR_STRING_PATTERN_MESSAGE = "There is a syntax error in the yaml content";
```

Comment:
I don't see any usages of these constants. can you please re-check?

Reply:
Let me remove these, this is related to changes we suppose to do log disable